app/graphql/client-only-query.ts

```diff
@@ -91,3 +97,20 @@ export const updateColorScheme = (client: ApolloClient<unknown>, colorScheme: st
     console.warn("impossible to update beta")
   }
 }
+
+export const feedbackModalShownScheme = (
+  client: ApolloClient<unknown>,
+  shown: boolean,
+) => {
+  try {
+    client.writeQuery<FeedbackModalShownQuery>({
+      query: FeedbackModalShownDocument,
+      data: {
+        __typename: "Query",
+        feedbackModalShown: shown,
+      },
+    })
+  } catch {
+    console.warn("feedback modal is not shown")
```

Comment:
message is not correct here. the error would arise if the apollo is unable to update the value